### PR TITLE
Updates to the User Guide document

### PR DESCRIPTION
**Pull request by jroberts91**
You can view current changes at:
https://github.com/jroberts91/main/blob/master/docs/UserGuide.adoc

### CODE REVIEW

**Comment by jroberts91** [comment]
Left with:
1. Removing image in Quickstart section
2. Updating command summary

**Comment by ChooJeremy** [comment]
Maybe try to incorporate some info and warning tags where appropriate? i.e. the current one's history has this tip for pressing the up and down arrow keys: https://github.com/CS2103-AY1819S1-F10-1/main/blob/master/docs/UserGuide.adoc

**Comment by ChooJeremy** [comment]
Also don't forget to change the 
By: Team SE-EDU Since: Jun 2016 Licence: MIT
at the top (can see developer guide for a sample)

File: docs/UserGuide.adoc [hunk 1]
```diff
@@ -1,4 +1,4 @@
-= AddressBook Level 4 - User Guide
+= Oasis - User Guide
```
Comment (KokJianYu) [inline docs/UserGuide.adoc]: Maybe all CAPS as it is an abbrevation?

Comment (RnardHa) [reply docs/UserGuide.adoc]: do it

**Comment by KokJianYu** [comment]
Maybe you could further break down the features tab to include both Employee features and Manager features.  So something like.

Features: 
>Employee Feature
>>feature 1,2,3 ...

>Manager Feature
>>feature 1,2,3

**Pull request merged**